lib/content_generator.rb

```diff
@@ -0,0 +1,128 @@
+# frozen_string_literal: true
+
+module ContentGenerator
+  module Vm
+    def self.location(location)
+      Option.locations.find { _1.display_name == location }.ui_name
+    end
+
+    def self.private_subnet(location, private_subnet)
+      private_subnet[:display_name]
+    end
+
+    def self.enable_ipv4(location, value)
+      location = LocationNameConverter.to_internal_name(location)
+      unit_price = BillingRate.rates.find { _1["resource_family"] == "IPv4" && _1["location"] == location && _1["resource_type"] == "IPAddress" }["unit_price"].to_f
```

Comment:
Possibly worth extracting a `BillingRate.unit_price_for(family, location, type)` method?

Reply:
Actually, we already have one. I updated this code to use it.